Src/BootCamp.Chapter/HealthCheckSimulator.cs

```diff
@@ -0,0 +1,67 @@
+﻿using BootCamp.Chapter.Logging;
+using System;
+
+namespace BootCamp.Chapter
+{
+    public static class HealthCheckSimulator
+    {
+        public static void Run()
+        {
+            Logger sessionLogger = SetupLogger();
+
+            Person user = new Person(); //We create our test subject
+            ConsolePrompter.WelcomeUser();
+            SetPersonalDetails(user, sessionLogger);
+
+            Console.WriteLine($"BMI equals: {BMICalculator.CalculateBMI(user, sessionLogger)}");
+        }
+
+        private static Logger SetupLogger()
```

Comment:
Extract this method to ``LoggerFactory``

Reply:
Extracted!